app/views/sign_up/registrations/new.html.erb

```diff
@@ -40,21 +40,35 @@
                  locals: { f: f, selection: @register_user_email_form.email_language } %>
     </fieldset>
 
+    <div class="margin-bottom-3">
+      <%= f.check_box :terms_accepted, { class: 'usa-checkbox__input',
+                                         required: true, aria: { invalid: false } }, true, false %>
+      <label for="user_terms_accepted" class="usa-checkbox__label">
+        <%= t('sign_up.terms') %>
+        <%= new_window_link_to(t('titles.rules_of_use'), MarketingSite.rules_of_use_url) %>
+      </label>
+      <div class="usa-error-message usa-error-message--with-icon display-if-invalid" role="alert">
+        <%= t('sign_up.agree_to_rules_of_use_and_continue') %>
+      </div>
+    </div>
```

Comment:
Should we only be showing this if `rules_of_use_enabled` config is `'true'`? (I expect we might also need to not add the corresponding JavaScript if disabled)